src/components/post-body.tsx

```diff
@@ -15,6 +17,7 @@ const PostBody = ({ content }: Props) => {
         )}
         dangerouslySetInnerHTML={{ __html: content }}
       />
+      <ReactUtterences repo={'gogaille/blog'} type={'pathname'} />
```

Comment:
We should add a space before the comment block and maybe also a tittle:

![image](https://user-images.githubusercontent.com/95120/109935125-89626900-7ccd-11eb-9e5e-6f07e142f243.png)

Reply:
For now it's not an issue, but one day we well have to improve the style of the comments